<launch>
	<node pkg="assignment_2_2024_client" type="target_client.py" name ="target_client" output "screen">
<launch>
